doc/Caliptra.md

```diff
@@ -409,7 +409,7 @@ Caliptra shall attest to the value of the owner key, enabling external verifiers
 
 The SoC may support a fuse bank for representing the hash of the owner's public key. If the SoC reports this value to Caliptra, Caliptra refuses to boot firmware unless the firmware was dual-signed by the key reported by SoC ROM's fuse registers.
 
-The owner key, when represented in fuses or in the FMC's alias certificate, is a SHA-384 hash of a structure that contains a list of owner public keys. This supports key rotation.
+The owner key, when represented in fuses or in the FMC's alias certificate, is an SHA384 hash of a structure that contains a list of owner public keys. This supports key rotation.
```

Comment:
Would've thought "a SHA384 hash" is preferred.

Reply:
@steph-morton to comment

Reply:
How is it pronounced? I assumed it would be pronounced spelled out as most acronyms are: "S-H-A". But if it is "shay" or "shaw" then @bluegate010 is correct and it should be "a SHA...".